drv.go

```diff
@@ -364,12 +364,12 @@ func (d *drv) openConn(P ConnectionParams) (*conn, error) {
 	}
 
 	c := conn{drv: d, connParams: P}
-	connString := P.StringNoClass()
+	connString := P.string(false, true)
```

Comment:
We don't need a new pool per-password, so we don't need the password in the connString, so we don't need the second parameter in P.string.

Reply:
Ok, in that case I don't think the second param is needed at all and `string` can always leave out the password. I'll push an update shortly.